apps/server/src/adapterSetup.ts

```diff
@@ -46,12 +56,12 @@ const mxAdapterMapObject = getMxAdapterMapObject({
 });
 
 // This is where you add adapters
-export const adapterMap = {
+export const adapterMap: Record<string, AdapterMap> = {
   ...templateAdapterMapObject,
   ...mxAdapterMapObject,
   ...sophtronAdapterMapObject,
   ...testAdapterMapObject,
 };
 
-export type Aggregator = keyof typeof adapterMap;
+export type Aggregator = keyof typeof adapterMap | string;
```

Comment:
Why the | string?

Reply:
Without this I was getting type errors while running some tests. I've just removed it, and it seems things are back to normal.